package.json

```diff
@@ -456,6 +456,7 @@
     "prepare": "npm run docs",
     "lint": "coffeelint src/ spec/",
     "code-docs": "codo && open docs/code/index.html",
+    "pretest": "npm lint",
```

Comment:
Should this be `npm run lint`?

Reply:
Also note `npm test` (AKA `npm run test`) is not run for Travis CI. Instead it calls only `sh build-package.sh`.
See https://github.com/joaomlneto/atom-beautify/blob/a766cbc34b7a41d2c17071c4c52332b5e1b4adba/.travis.yml#L12 

Fortunately, `coffeelint` is already called by https://github.com/joaomlneto/atom-beautify/blob/a766cbc34b7a41d2c17071c4c52332b5e1b4adba/build-package.sh#L20-L31 so this is OK.

Reply:
> Should this be npm run lint?

That's correct! 😨